tests/Mono.Android-Tests/Runtime-Microsoft.Android.Sdk/Mono.Android.NET-Tests.csproj

```diff
@@ -22,10 +22,10 @@
     <AndroidUseNegotiateAuthentication>true</AndroidUseNegotiateAuthentication>
     <!--
       TODO: Fix excluded tests
-      For AOT, InetAccess excluded due to: https://github.com/dotnet/runtime/issues/56315
+      For $(EnableLLVM), InetAccess excluded due to: https://github.com/dotnet/runtime/issues/56315
     -->
     <ExcludeCategories>DotNetIgnore</ExcludeCategories>
-    <ExcludeCategories Condition=" '$(RunAOTCompilation)' == 'true' ">$(ExcludeCategories):InetAccess</ExcludeCategories>
+    <ExcludeCategories Condition=" '$(EnableLLVM)' == 'true' ">$(ExcludeCategories):InetAccess</ExcludeCategories>
```

Comment:
Since 0084162605ad0785c961a79be1ef8a0305e19d7f, this `Condition` wasn't actually doing anything. We weren't passing in `RunAOTCompilation=true` because that  is the default value for `Release` builds.

`InetAccess` seems to crash the tests when `EnableLLVM=true`, however... So let's keep the condition for that case.

Reply:
Can you please also open an issue for the `InetAccess` test crashes with LLVM enabled?